kevpy/SendIT
# #161773708 write and setup tests

#### What does this PR do?
Set up and write tests. This includes setting up badges for travis and coveralls

#### Description of Task to be completed?
Write tests for the following endpoints:
`/api/v1/parcels` `POSTt`
`/api/v1/parcels` `GET`
`/api/v1/parcels/<parcel_id>` `GET`
`/api/v1/<parcel_id>/cancel` `PUT`
`/api/v1/user/<user_id>/parcels` `GET`

#### How should this be manually tested?
Clone this repo
Set up a Flask environment by running pip install on requirements.txt on a virtual environment.
Run py.test 
This should show all tests running

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#161773708
